<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSnippets</class>
 <widget class="QDialog" name="DialogSnippets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snippets selector</string>
  </property>
  <widget class="QToolButton" name="btnEq">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDiff">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSim">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
